ddtrace/ddtrace.go

```diff
@@ -108,4 +108,7 @@ type StartSpanConfig struct {
 	// Tags holds a set of key/value pairs that should be set as metadata on the
 	// new span.
 	Tags map[string]interface{}
+
+	// Force-set the SpanID, rather than use a random number.
+	SpanID *uint64
```

Comment:
It's fine to have a regular `uint64` here. We don't need a pointer. It's safe to assume that if the value is `0` it wasn't intended for the span ID to be zero either. I mean, the [span type](https://github.com/DataDog/dd-trace-go/blob/v1/ddtrace/tracer/span.go#L46) itself work on the same assumption.